runtime/Go/antlr/tree.go

```diff
@@ -28,7 +28,7 @@ type SyntaxTree interface {
 type ParseTree interface {
 	SyntaxTree
 
-	Accept(Visitor ParseTreeVisitor) interface{}
+	Accept(visitor ParseTreeVisitor)
```

Comment:
Sure, I've put up https://github.com/cyberfox/expressions for now, and I'd be happy to mix it into a catalog of examples.  I did integrate returns and actions for a bit, while using the visitor at the same time.  (My QuotedLiteral sliced the quotes off in the grammar, but I realized it was easier and less prone to weird compile errors to just write it as a visitor.)

For what it's worth (as I bet you guessed), I first built a grammar for Java, because that's what I was targeting for work.  I started stripping out actions as I improved my Java visitors, and eventually I was left with only a few and a _bunch_ of returns, when...

I realized there was a Go target, and I've had a goal to improve my Go skills, and so I started making it work for both languages.  Doing that from one grammar forced me to strip out all the actions and returns.  It also forced me to make the Visitor ANTLR code work. :)  So 'prototype a grammar with actions, args, and returns' and then moving to visitors afterwards was pretty much exactly the process I used.